### DISCUSSION TITLE
Welcome to yahooquery Discussions!

### DISCUSSION THREAD

**Discussion by dpguthrie** [dpguthrie/yahooquery#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bmazeraski** [comment]
Doug... I'm interested in accessing some of the 'Premium' content from Yahoo Finance using yahooquery, but it looks like the 'Premium' membership has been replaced with either the 'Lite' version ($25/month) or the 'Essential' version ($35/month). Will the 'Lite' version satisfy all of the yahooquery 'Premium' endpoints, or do I need to sign up for Yahoo's 'Essential' version. Please advise.
